tests/unit/multi.tcl

```diff
@@ -631,6 +635,24 @@ start_server {tags {"multi"}} {
         close_replication_stream $repl
     } {} {needs:repl}
 
+    test {MULTI propagation of SCRIPT FLUSH} {
```

Comment:
We didn't mention `script flush` command (or I forgot about it). Sure that we can drop the 'MULTI/EXEC' part and only test about `script flush`, but it does no harm adding the 'MULTI/EXEC', which can cover more cases, doesn't it?

Reply:
@MeirShpilraien if you have any clues about it, please let us know.

Reply:
now that it's a trivial command (no propagation), in theory there's no need to test it neither here no there (with / without MULTI).
back when SCRIPT LOAD had special (non trivial) handling for propagation, it made sense to test it both ways.
i don't mind keeping the MULTI (it just looks odd since i didn't understand why should we), but i also don't mind dropping this test completely, or just keep it simpler and remove the MULTI.

Reply:
Actually if we decide not to change the behaviour of `script flush` any more, we could just drop this case. Otherwise we'd better keep this case? Maybe it looks weird but it's for robustness I think?

Reply:
fine. let's keep it.